### PR TITLE
184 unique value

### LINKED ISSUE #184: [Filters] Remove `This column has one unique value` pop-up
### context
In `faceted search` when selecting a column with one value, a message appears directly below the column name that says `This column has one unique value` -- this should be removed per clients request; all other messages are okay.

### approach
Delete message / editor leftover

### PR DESCRIPTION
**Pull request by coyle5280**
This closes #184 

# Context
This PR modifies `histogram.js`, `histogram.jst.ejs`, and `selection.jst.ejs`. In `histogram.js` the code was modifies to allow a histogram with one value to be displayed.  The value display is updated with the signle value and the "brush bars" are removed. `<% if (status == 'only') { %><div class="<%- status %>">This column has one unique value</div><% } %>` was removed from `histogram.jst.ejs`, and `selection.jst.ejs`.

# Acceptance
- [x] A column filter with a single numerical or date value will display as a histogram.
- [x] A single value histogram with have NO "brush bars".
- [x] Single value histogram does not allow editing of input fields. 
- [x] Multi value numerical histograms function as always. No Changes.
- [x] Multi value Date histograms function as always. No Changes.

### CODE REVIEW

File: lib/assets/javascripts/cartodb/table/menu_modules/filters/histogram.js [hunk 1]
```diff
@@ -740,17 +726,17 @@
...
+            .attr("id", "clip-" + id)
+            .append("rect")
+            .attr("width", width)
+            .attr("height", height);
+
+          g.selectAll(".bar")
+            .data(['background', 'foreground'])
+            .enter().append("path")
+            .attr("class", function(d) { return d + " bar"; })
+            .data([group, group])
```
Comment (gfiorav) [inline lib/assets/javascripts/cartodb/table/menu_modules/filters/histogram.js:739]: Have you changed the indentation on this?

File: lib/assets/javascripts/cartodb/table/menu_modules/filters/histogram.js [hunk 2]
```diff
@@ -769,10 +755,12 @@
           g.selectAll("text")
             .attr("fill", "#d8d8d8");
 
-        // Initialize the brush component with pretty resize handles.
-        var gBrush = g.append("g").attr("class", "brush").call(brush);
-        gBrush.selectAll("rect").attr("height", height);
-        gBrush.selectAll(".resize").append("path").attr("d", resizePath);
+          // Initialize the brush component with pretty resize handles.
```
Comment (gfiorav) [inline lib/assets/javascripts/cartodb/table/menu_modules/filters/histogram.js:758]: Same here: is the indentation inconsistent?

**Review by gfiorav — COMMENTED** [review]
Left some comments

**Comment by alissarose** [comment]
@coyle5280 - please respond to guido's questions above

on my side, this looks good in the VM

File: lib/assets/javascripts/cartodb/table/menu_modules/filters/histogram.js (hunk 2 above)
Comment (coyle5280) [reply lib/assets/javascripts/cartodb/table/menu_modules/filters/histogram.js:758]: Yes. Same here. Just a small adjustment to make it line up better.

File: lib/assets/javascripts/cartodb/table/menu_modules/filters/histogram.js (hunk 1 above)
Comment (coyle5280) [reply lib/assets/javascripts/cartodb/table/menu_modules/filters/histogram.js:739]: Yes. I Adjusted it to be more consistent with the rest of the document

**Review by gfiorav — APPROVED** [review]

**Pull request merged**